config/job.yaml

```diff
@@ -0,0 +1,55 @@
+# Job Configuration
+
+job:
+    id: 
+    cluster_configuration:
```

Comment:
can we put in some defaults? 
- vm size - d2_v2
- size - 5
- size low pri - 0
- docker_repo: [commented for smart default]
- custom_scripts: [commented]